src/pages/CreateEvent/EventForm.tsx

```diff
@@ -163,230 +232,373 @@ const EventForm: React.FC = () => {
-                placeholder="Enter judge name"
-                onChange={(e) => handleJudgeChange(index, e.target.value)}
-              />
-              {index > 0 && (
-                <button
-                  type="button"
-                  className="remove-judge-button"
-                  onClick={() => handleRemoveJudge(index)}
-                >
-                  X
-                </button>
-              )}
+          <label htmlFor="judges">Judges<sup>*</sup></label>
+          {fields.map((field, index) => (
+            <div
+              key={field.id}
+              className="justify-between flex-col flex whitespace-nowrap my-3"
+            >
+              <div className="flex gap-2 items-center">
+                <label className="mr-2">Judge #{index + 1}</label>
+                <input
+                  type="text"
+                  {...register(`judges.${index}.firstName`, {
+                    required: "First name is required",
+                  })}
+                  placeholder="Enter first name"
+                  className={`w-3/6 mt-2 mb-1 p-2 border border-black rounded ${
+                    errors.judges?.[index]?.firstName ? "error" : ""
+                  }`}
+                />
+
+                <input
+                  type="text"
+                  {...register(`judges.${index}.lastName`, {
+                    required: "Last name is required",
+                  })}
+                  placeholder="Enter last name"
+                  className={`w-3/6 mt-2 mb-1 p-2 border border-black rounded ${
+                    errors.judges?.[index]?.lastName ? "error" : ""
+                  }`}
+                />
+                {fields.length > 1 && (
+                  <button
+                    type="button"
+                    className="ml-4 text-red-500"
+                    onClick={() => remove(index)}
+                  >
+                    X
+                  </button>
+                )}
+              </div>
+
+              <div className="flex flex-col">
+                {errors.judges?.[index]?.firstName && (
+                  <p className="error">
+                    {errors.judges[index].firstName.message}
+                  </p>
+                )}
+                {errors.judges?.[index]?.lastName && (
+                  <p className="error">
+                    {errors.judges[index].lastName.message}
+                  </p>
+                )}
+              </div>
             </div>
           ))}
-          {judges.length < 4 && (
+
+        {fields.length <3 && (
+          <div className="flex justify-end mt-2">
             <button
               type="button"
-              className="add-judge-button"
-              onClick={handleAddJudge}
+              className="px-4 py-2 rounded-[10px] bg-MVP-light-blue border-[3px] border-r-[5px] border-b-[5px] border-MVP-black text-MVP-black"
+              onClick={() => append({ firstName: "", lastName: "" })}
             >
```

Comment:
if you have time you can disabled the add judge button until the organizer fills out the first and last name of the previous judge first

Reply:
Done